reader.go

```diff
@@ -1464,8 +1466,9 @@ func (this *PdfReader) rebuildContentStream(content *PdfValue) ([]byte, error) {
 			encoded := stream
 			// the -3 strips the end of data marker
 			decodedBytes, err := ioutil.ReadAll(ascii85.NewDecoder(bytes.NewBuffer(encoded[:len(encoded)-3])))
+			// Sometimes there's an error decoding ascii85 but appending the packing slip will still work so we log it and move on
 			if err != nil {
-				return nil, err
+				log.FromContext(ctx).Infof("error rebuilding content from stream: %s", err)
```

Comment:
Is there any info we can attach to this to make the long more useful?

Also, please include the error in the fields instead of the message:
```suggestion
				log.FromContext(ctx).WithError(err).Info("error rebuilding content from stream")
```

Reply:
In the Packingslip handlers.go file I was going to include the return id there
```
	ctx := log.WithFields(r.Context(), logrus.Fields{
		"scope": "PackingSlip",
		"id":    id,
	}
```)

Reply:
Ok, that is fine.